nflow-jetty/src/main/java/io/nflow/jetty/mapper/ErrorResponse.java

```diff
@@ -0,0 +1,17 @@
+package io.nflow.jetty.mapper;
+
+import io.nflow.engine.model.ModelObject;
+
+public class ErrorResponse extends ModelObject {
```

Comment:
Should this be shared with the other rest api to keep the api consistent?
No need to use it now, but moving this to the shared project would be wise?